styles/globals.css

```diff
@@ -13,98 +19,22 @@ html {
   overflow-x: hidden;
 }
 
-.ExternalLayout_wrapper {
-  width: 100%;
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  min-height: 100vh;
-}
-
-.UsecaseWrapper_divider {
-  max-width: 600px;
-  width: 100%;
-}
-
-.UsecaseWrapper_description {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  padding: 32px;
-}
-
-.ExternalLayout_main {
-  width: 100%;
-  padding: 8px 42px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-}
-
-.UsecaseWrapper_wrapper {
-  padding: 0 16px;
-  max-width: 600px;
-  flex: 1 1;
-}
-
-.UsecaseWrapper_wrapper.full {
-  max-width: initial;
-}
-
-.UsecaseWrapper_title {
-  width: 100%;
-  font-weight: 500;
-  font-size: 32px;
-}
-
-.UsecaseWrapper_helper {
-  margin-bottom: 1rem;
-  text-align: left;
-  opacity: 0.72;
-  font-size: 16px;
-}
-
-.UsecaseWrapper_notes {
-  font-size: 14px;
-  font-weight: 400;
-  margin-top: 1rem;
-  margin-bottom: 0.75rem;
-  text-align: left;
-}
-
-.UsecaseWrapper_notes li {
-  margin-bottom: 12px;
+a {
```

Comment:
Let's try to get ride of the CSS variables (it causes duplicates with the MUI Theme).

Can we move the global styles to [GlobalStyles](https://mui.com/material-ui/customization/how-to-customize/#4-global-css-override) ?

Reply:
Would you mind if we did that in a separate PR? We can remove `globals.css` and `web-scraping.module.css` and move everything to `GlobalStyles` and `sx`, but right now I would like to focus on the playground demo which has a Wednesday deadline. I can make a ticket for the styles clean-up.

Reply:
Sure, sounds good 😃

Reply:
Alright, here is a ticket: https://github.com/orgs/fingerprintjs/projects/7/views/16?pane=issue&itemId=29475404
Thanks!